lib/integration/scenarios/Test/Integration/Scenario/API/Shelley/TransactionsNew.hs

```diff
@@ -895,7 +895,11 @@ spec = describe "NEW_SHELLEY_TRANSACTIONS" $ do
             (Link.createUnsignedTransaction @'Shelley wa) Default payload
         verify rTx
             [ expectResponseCode HTTP.status403
-            , expectErrorMessage errMsg403MinUTxOValue
+            , expectErrorInfo $ flip shouldSatisfy $ \case
```

Comment:
I wonder if we could define wrapper function in DSL that will have structure :
```
            wrapper APIError = expectErrorInfo $ flip shouldSatisfy $ \case
                APIError {} ->
                    True
                _anythingElse ->
                    False
```
It seems this is repeated all around.